### DISCUSSION TITLE
Where do exported recipes go?

### DISCUSSION THREAD

**Discussion by Sylvanling** [shedaniel/RoughlyEnoughItems#905] [Q & A] [answered]
When I export a recipe it tells me to check the "rei_exports" folder, but I can't find this folder anywhere. Where is it?

**Comment by shedaniel** [comment, marked as answer]
It is in the .minecraft folder

---

If this answer helped you, please mark this as the answer.

**Reply by Sylvanling** [reply]
No it isn't. 
![image](https://user-images.githubusercontent.com/78134002/173197089-44625405-1f2a-4025-8e19-7b8f701eb63e.png)

**Reply by shedaniel** [reply]
Are you sure that's your game folder? In addition, I think there was a fix for recipe exporting a couple days ago affecting some users on Windows, maybe you can try the latest update.

**Reply by Sylvanling** [reply]
Updating fixed it, thank you for the help!
